src/HealthChecks.Publisher.CloudWatch/HealthChecks.Publisher.CloudWatch.csproj

```diff
@@ -0,0 +1,15 @@
+<Project Sdk="Microsoft.NET.Sdk">
+
+  <PropertyGroup>
+    <TargetFrameworks>$(NetFrameworkVersion)</TargetFrameworks>
+    <PackageTags>$(PackageTags);Publisher;AWS CloudWatch</PackageTags>
+    <Description>HealthChecks.Publisher.CloudWatch is the health check publisher for AWS CloudWatch.</Description>
+    <Version>$(HealthCheckPublisherCloudWatch)</Version>
```

Comment:
Where this come from?

Reply:
HealthChecks.Publisher.ApplicationInsights.csproj

Reply:
I mean I don't see HealthCheckPublisherCloudWatch property definition.

Reply:
See https://github.com/Xabaril/AspNetCore.Diagnostics.HealthChecks/blob/master/build/dependencies.props#L41-L44

Reply:
Feel free to change... I just trying to contribute in someway

Reply:
Oh ok... Can you do this? I just left my laptop right now

Reply:
Tomorrow or later.

Reply:
done